packages/astro/src/integrations/index.ts

```diff
@@ -61,6 +61,25 @@ function getLogger(integration: AstroIntegration, logger: Logger) {
 	return integrationLogger;
 }
 
+const serverEventPrefix = 'astro-dev-toolbar';
+
+export function getToolbarServerCommunicationHelpers(server: ViteDevServer) {
```

Comment:
Should we document each function, to give some context of what they do and when to use them?

Reply:
I fear that for most of those functions, I'll just end up saying the function name twice. Like, "onAppInit is fired when the app is init"